### PR TITLE
NEW Rename delete button to archive button. Update all references.

### LINKED ISSUE #299: Add "more actions" item to archive block individually
Subtask from https://github.com/dnadesign/silverstripe-elemental/issues/290

* "Archive" option is in more actions menu
* GraphQL mutation given the element ID deletes the current draft version of the element (we assume that deleting would be an OOB CRUD scaffolded mutation operation)
* Apollo cache updates to update the state of the Element (ideally) or fallback to allowing the "read all blocks" query to re-run and reload all blocks

### PR DESCRIPTION
**Pull request by raissanorth**
Renames delete button to archive button. This aligns with the designs:

image

For consistency the GraphQL mutation has been renamed as well.

In combination with b9d89f375f94a1d750a this resolves #299

### CODE REVIEW

File: client/src/components/ElementEditor/Header.js [hunk 1]
```diff
@@ -27,19 +27,26 @@ class Header extends Component {
...
     event.stopPropagation();
 
-    const { id, actions: { handleDeleteBlock } } = this.props;
+    const { id, isPublished, actions: { handleArchiveBlock } } = this.props;
 
-    const deleteMessage = i18n._t(
+    let archiveMessage = i18n._t(
       'ElementHeader.CONFIRM_DELETE',
-      'Are you sure you want to delete this block?'
+      'Are you sure you want to sent this block to the archive?'
```
Comment (ScopeyNZ) [inline client/src/components/ElementEditor/Header.js]:
```diff
-sent
+send
```

**Comment by raissanorth** [comment]
@ScopeyNZ I don't like the non-alphabetical order either. But again, I went with the designs:
image
 Perhaps something to discuss with @newleeland tomorrow.

**Comment by robbieaverill** [comment]
I think it’s ok, the important actions (save and publish) should be first in the list (in my opinion)

**Comment by robbieaverill** [comment]
@raissanorth can you fix the CI builds?

**Pull request merged**